src/styles/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat($columnsAmount, 2);
```

Comment:
The value `2` in `grid-template-columns: repeat($columnsAmount, 2);` seems to be incorrect or incomplete. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `grid-template-columns` property should have a unit of measurement like `fr`, `px`, `%`, etc. Please provide a valid unit or use a different value that aligns with the design requirements.